components/brave_new_tab_ui/components/default/braveToday/cards/cardOptIn.tsx

```diff
@@ -14,27 +14,31 @@ type Props = {
   onDisable: () => unknown
 }
 
+const descriptionTwoTextParts = getLocaleWithTag('braveNewsIntroDescriptionTwo')
+
 export default function IntroCard (props: Props) {
   const introElementRef = React.useRef(null)
-  const descriptionTextParts = React.useMemo(() => {
-    return getLocaleWithTag('braveTodayIntroDescription')
-  }, [])
   return (
     <Card.Intro ref={introElementRef}>
       <Card.Image src={BraveTodayLogoUrl} />
-      <Card.Title>{getLocale('braveTodayIntroTitle')}</Card.Title>
-      <Card.Paragraph>
-        {descriptionTextParts.beforeTag}
-        <a href={'https://brave.com/privacy/browser/'}>
-          {descriptionTextParts.duringTag}
-        </a>
-        {descriptionTextParts.afterTag}
-      </Card.Paragraph>
+      <Card.Title>{getLocale('braveNewsIntroTitle')}</Card.Title>
+      <div>
+        <Card.Paragraph>
+          {getLocale('braveNewsIntroDescription')}
+        </Card.Paragraph>
+        <Card.Paragraph>
+          {descriptionTwoTextParts.beforeTag}
+          <a href={'https://brave.com/privacy/browser/'}>
```

Comment:
Not sure what our style guide says about `{}` on urls?

```suggestion
          <a href='https://brave.com/privacy/browser/'>
```

Reply:
Our linter used to enforce props should be surrounded by {} - I thought for all JSX elements, but it doesn't seem to be enforced either way atm. I'm not sure where the team wants to fall on this.